client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -59,84 +56,136 @@ import {
 	isRequestingSites,
 	getSiteAdminUrl,
 } from 'calypso/state/sites/selectors';
-import {
-	getSelectedSite,
-	getSelectedSiteId,
-	getSelectedSiteSlug,
-} from 'calypso/state/ui/selectors';
+import { getSelectedSite, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import './style.scss';
 
 /**
  * Module variables
  */
 const SHORT_LIST_LENGTH = 6;
 const SEARCH_RESULTS_LIST_LENGTH = 12;
-const VISIBLE_CATEGORIES = [ 'new', 'popular', 'featured' ];
-const VISIBLE_CATEGORIES_FOR_RECOMMENDATIONS = [ 'new', 'popular' ];
```

Comment:
Wondering if these shouldn't be removed, what's the rationale here?

Reply:
On the `trunk` version those constants are only being used by the function `getVisibleCategories`which is not being used anywhere. 

While doing the refactor I removed the function and those unused constants.
https://github.com/Automattic/wp-calypso/blob/ddf71bb3fd5891b3ce84380da5ccbda1bce28ca7/client/my-sites/plugins/plugins-browser/index.jsx#L123-L129

Reply:
That's great, thanks!